### PR TITLE
+3 icons

### PR DESCRIPTION
**Pull request by M-Gholami**
# Description
3 icons added.

### Added

Mead Mate (`com.dawsonsoftware.meadmate`)
Nobitex (`market.nobitex`)
Bank Mehr ~~ بانک مهر (`com.ada.mbank.mehr`)

## Contributor's checklist
- [x] I followed [the Lawnicons guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: app/assets/appfilter.xml [hunk 1]
```diff
@@ -881,6 +881,7 @@
   <item component="ComponentInfo{com.bandlab.bandlab/com.bandlab.navigation.entry.NavigationActivity}" drawable="bandlab" name="BandLab" />
   <item component="ComponentInfo{com.bandsintown/com.bandsintown.activity.SplashScreen}" drawable="bandsintown" name="Bandsintown" />
   <item component="ComponentInfo{com.bandsintown/com.bandsintown.SplashScreen}" drawable="bandsintown" name="Bandsintown" />
+  <item component="ComponentInfo{com.ada.mbank.mehr/com.ada.mbank.component.SplashActivity}" drawable="bank_mehr" name="Bank Mehr ~~ بانک مهر" />
```
Comment (x9136) [inline app/assets/appfilter.xml:884]: How is it more convenient to read? Is there a difference?

1. Bank Mehr ~~ بانک مهر
2. بانک مهر ~~ Bank Mehr

File: svgs/mead_mate.svg
Comment (x9136) [inline svgs/mead_mate.svg:1]: Screenshot 2024-09-13 at 16 09 24
The icon size is a little small and the lines in the highlighted area are too close to each other.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: app/assets/appfilter.xml (hunk 1 above)
Comment (M-Gholami) [reply app/assets/appfilter.xml:884]: There's not much difference here, I'll change to 2 option if that's more consistent with the rest of icons.

Comment (x9136) [reply app/assets/appfilter.xml:884]: Yes, option 2 is better suited to the current format. If option 1 were more convenient, I would change the format for languages that are read from right to left.

File: svgs/bank_mehr.svg
[the remark this answers is not in the document]
Comment (M-Gholami) [reply svgs/bank_mehr.svg:1]: I think connecting those lines would simplify the icon too much.
I'll try to increase the clearance between them.
image

**Review by x9136 — APPROVED** [review]

**Pull request merged**